webapp/test/test-runs.html

```diff
@@ -35,15 +35,15 @@
           let interval = setInterval(() => {
             if (predicate()) {
               clearInterval(interval);
-              resolve(true);
+              resolve();
             }
           }, 0);
         });
       }
 
       setup(() => {
         sandbox = sinon.sandbox.create();
-        sandbox.stub(window, 'fetch', () => Promise.resolve(new Response(fetches[0])));
+        sandbox.stub(window, 'fetch', () => Promise.resolve(new Response(fetches)));
```

Comment:
Why did you remove `[0]`?

Reply:
It's fetching `/api/runs` which returns an array, so the result should be the array. Not sure why it was a single item originally.